applications/GeoMechanicsApplication/tests/fixed_spatial_variation/MaterialParameters.json

```diff
@@ -0,0 +1,35 @@
+{
+   "properties": [{
+      "model_part_name":         "PorousDomain.soil",
+      "properties_id":           1,
+      "Material": {
+          "constitutive_law": {
+              "name"             :  "GeoLinearElasticPlaneStrain2DLaw" 
+          },
+          "Variables": {
+              "IGNORE_UNDRAINED"         :  false,
+              "YOUNG_MODULUS"            :  10000,
+              "POISSON_RATIO"            :  0.2,
+              "DENSITY_SOLID"            :  2650,
+              "DENSITY_WATER"            :  1000,
+              "POROSITY"                 :  0.35,
+              "BULK_MODULUS_SOLID"       :  1.0e9,
+              "BULK_MODULUS_FLUID"       :  2.0e6,
+              "PERMEABILITY_XX"          :  1.157e-12,
+              "PERMEABILITY_YY"          :  1.157e-12,
+              "PERMEABILITY_XY"          :  0.0,
+              "DYNAMIC_VISCOSITY"        :  1e-3,
+              "THICKNESS"                :  1.0,
+              "BIOT_COEFFICIENT"         :  1.0,
+              "RETENTION_LAW"                    : "SaturatedLaw",
+              "SATURATED_SATURATION"             :  1.0,
+              "RESIDUAL_SATURATION"              :  0.06203,
+              "VAN_GENUCHTEN_AIR_ENTRY_PRESSURE" :  2.561,
+              "VAN_GENUCHTEN_GN"                 :  1.377,
+              "VAN_GENUCHTEN_GL"                 :  1.25,
+              "MINIMUM_RELATIVE_PERMEABILITY"    :  0.0001
```

Comment:
These have no meaning for SaturatedLaw. Sadly they will be completed from defaults afterwards, but I think we should not include non-used information in new tests. Also THICKNESS is redundant.

Reply:
Removed